gallery/src/components/demo-more-info.js

```diff
@@ -0,0 +1,93 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../../../src/state-summary/state-card-content.js';
+import '../../../src/dialogs/more-info/controls/more-info-content.js';
+import '../../../src/components/ha-card.js';
+
+
+class DemoMoreInfo extends PolymerElement {
+  static get template() {
+    return html`
+        <style>
+          :host {
+            display: flex;
+            align-items: start;
```

Comment:
default value

Reply:
Without I get this:

![image](https://user-images.githubusercontent.com/1444314/43065982-bd769212-8e63-11e8-8e7e-44ab0125cbd7.png)

Reply:
(the more info card is stretched to the height of the `<pre>`)